packages/framework/src/Model/Order/Order.php

```diff
@@ -923,4 +923,13 @@ public function isCancelled()
     {
         return $this->status === OrderStatus::TYPE_CANCELED;
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Order\OrderPriceCalculation $orderPriceCalculation
+     */
+    public function calculateTotalPrice(OrderPriceCalculation $orderPriceCalculation)
```

Comment:
What is even greather - when we remove `OrderCreationService`, we can make `calculateTotalPrice` protected. Because it doesn't make sense to separate use cases of create-calculate, and edit-calculate